### PR TITLE
zh-translation: docs/root/intro/arch_overview/other_features/local_ra…

### PR DESCRIPTION
**Pull request by scilla0531**
…te_limiting.rst

ref: https://github.com/cloudnativeto/envoy/issues/38

For an explanation of how to fill out the fields, please see the relevant section
in [PULL_REQUESTS.md](https://github.com/envoyproxy/envoy/blob/master/PULL_REQUESTS.md)

Commit Message:
Additional Description:
Risk Level:
Testing:
Docs Changes:
Release Notes:
[Optional Runtime guard:]
[Optional Fixes #Issue]
[Optional Deprecated:]

### CODE REVIEW

File: docs/root/intro/arch_overview/other_features/local_rate_limiting.rst [hunk 1]
```diff
@@ -1,16 +1,10 @@
...
 
-Envoy supports local (non-distributed) rate limiting of L4 connections via the
-:ref:`local rate limit filter <config_network_filters_local_rate_limit>`.
+Envoy 通过 :ref:`本地限速过滤器 <config_network_filters_local_rate_limit>` 支持基于 L4 连接的本地（非分布式）限速。
 
-Envoy additionally supports local rate limiting of HTTP requests via the
-:ref:`HTTP local rate limit filter <config_http_filters_local_rate_limit>`. This can
-be activated globally at the listener level or at a more specific level (e.g.: the virtual
-host or route level).
+Envoy 还通过 :ref:`HTTP 本地限速过滤器 <config_http_filters_local_rate_limit>` 支持基于 HTTP 请求的本地限速。可以全局激活该功能，在监听器级别或者其他特定级别（例如，虚机或路由级别）
```
Comment (helight) [inline docs/root/intro/arch_overview/other_features/local_rate_limiting.rst]:
```suggestion
Envoy 还通过 :ref:`HTTP 本地限速过滤器 <config_http_filters_local_rate_limit>` 支持基于 HTTP 请求的本地限速。可以在在监听器级别或者其他特定级别（例如，虚机或路由级别）全局激活该功能。
```

**Review by helight — COMMENTED** [review]
就1个地方感觉这样会比较顺畅一点。

**Comment by scilla0531** [comment]
@helight  已经修改，谢谢哈

**Comment by helight** [comment]
LGTM
/approve

**Pull request merged**